tests/integration/api/controllers/login.spec.js

```diff
@@ -78,6 +78,11 @@ const setupTokenLoginSettings = (configureAppUrl = false) => {
     .then(() => utils.addTranslations('en', { login_sms: 'Instructions sms' }));
 };
 
+const setupOidcLoginSettings = async (clientId) => {
+  const settings = { oidc_provider: { client_id: clientId } };
+  return await utils.updateSettings(settings, { ignoreReload: true });
```

Comment:
suggestion: can you add a call to `utils.revertCustomSettings` in the `afterEach`?  Just want to make sure any settings changes do not accidentally leak across tests.